astro/src/content/blog/announcing-fusionauth-1-62.mdx

```diff
@@ -0,0 +1,96 @@
+
+## Pre-Verified Identities
+
+Traditional verification happens after an account is created. Users create the account, FusionAuth then sends a verification email, and you hope the user completes it. Maybe there was a typo, or maybe a user is squatting on an identity. Either way, this creates an admin burden of managing unverified accounts, and introduces account takeover risks.
+
+Identity Pre-Verification ensures email addresses or phone numbers are verified during registration, before the account is created.
+
+### How It Works
+
+1. Users provide their info in a registration form
+2. They verify via a code or link sent immediately
+3. No account is created until verification succeeds
+
+This should cut down on the administrative burden of cleaning up “ghost” accounts, and everyone could use a bit less time spent doing “busy work.”
+
+## MFA Lambda: Smarter Authentication Decisions
+
+The MFA Lambda gives you programmatic control over when multi-factor authentication is required during login.
+
+### The Problem We're Solving
+
+FusionAuth's contextual MFA is powerful but limited to FusionAuth-defined contexts like device trust and suspicious login detection. You couldn't easily require MFA for your "Administrators" group, challenge users from unexpected locations, or integrate with your threat intelligence systems. Workarounds existed, but they were complex.
+
+### Where It Runs
+
+The MFA Lambda runs during login and password changes, making intelligent decisions about whether MFA is needed based on context. It can also be used during step-up authentication. Your application can now ask FusionAuth whether to challenge for MFA rather than deciding entirely on its own.
+
+### MFA Lambda vs. Step-Up Auth
+
+These might seem similar, but they're different:
+
+MFA Lambda is logic embedded in FusionAuth that runs during every login, making intelligent decisions about whether MFA is needed based on context—user groups, geography, time, external APIs, whatever you need.
+
+[Step-Up Auth](/docs/lifecycle/authenticate-users/multi-factor-authentication#step-up-auth) is embedded in your application, triggered by your code when users perform sensitive actions after login. Step-up authentication was introduced in [FusionAuth 1.26.0](/docs/release-notes/archive#new-13).
+
+**MFA Lambda**: smarter decisions during login.
+
+**Step-up Auth**: protecting sensitive actions after login.
+
+### What You Can Do
+
+Your lambda receives context about the login attempt—user, groups, roles, IP, device trust status, suspicious login detection (and geography if you have enabled Advanced Threat Detection)—and tells FusionAuth whether to require MFA or skip it. You can even make API calls from within the lambda. As long as they respond before the timeout, you can leverage existing business rules to determine whether or not to challenge for MFA.
```

Comment:
This is the first I'm hearing of a timeout -- maybe I missed it in the existing doc? If not, we should add something about that. All I've seen is vague mentions that latency will impact users, so you shouldn't do much in the way of API calls in an MFA lambda.